pynecone/components/forms/switch.py

```diff
@@ -3,7 +3,7 @@
 
 from pynecone.components.component import EVENT_ARG
 from pynecone.components.libs.chakra import ChakraComponent
-from pynecone.vars import Var
+from pynecone.var import Var
```

Comment:
We recently merged a PR renaming the `var.py` file to `vars.py` so we will need to update this